src/traces/funnelarea/attributes.js

```diff
@@ -55,15 +55,15 @@ module.exports = {
     }),
 
     texttemplate: texttemplateAttrs({editType: 'plot'}, {
-        keys: ['label', 'color', 'value', 'percent', 'text']
+        keys: ['label', 'color', 'value', 'text', 'percentInitial', 'percentTotal', 'percentPrevious']
     }),
 
     hoverinfo: extendFlat({}, plotAttrs.hoverinfo, {
         flags: ['label', 'text', 'value', 'percent', 'name']
     }),
 
     hovertemplate: hovertemplateAttrs({}, {
-        keys: ['label', 'color', 'value', 'percent', 'text']
+        keys: ['label', 'color', 'value', 'text', 'percentInitial', 'percentTotal', 'percentPrevious']
```

Comment:
Also here:
Those should be fixed in `funnel` not `funnelarea`.

Reply:
Fixed in 8c339df8c17b60af28ad38831924b03ec1141acc